File: echo.py

import sys
# Set up the input and output files
sys.stdin = open('A-small-2-attempt0.in.txt', 'r')
sys.stdout = open('A2sample.out', 'w')
# Read in T
total_cases = int(raw_input())
for case_number in xrange(1, total_cases + 1):
  # Read in C and W
  copies, word = raw_input().split()
  copies = int(copies)
  word = " " + word
  # Output the result
  print 'Case #%d: %s' % (case_number, word * copies)

  

File: echo.cc

#include <iostream>
#include <cstdio>
#include <algorithm>
using namespace std;
int main() {
  // Set up the input and output files
  freopen("sample.in", "r", stdin);
  freopen("sample.out", "w", stdout);
  // Read in T
  int total_cases;
  cin >> total_cases;
  for (int case_number = 1; case_number <= total_cases; case_number++) {
    // Read in C and W
    int copies;
    string word;
    cin >> copies >> word;
    // Output the result
    printf("Case #%d:", case_number);
    for (int i = 0; i < copies; i++) {
      printf(" %s", word.c_str());
    }
    printf("\n");   // Move to the next line
  }
}